### ISSUE TITLE
https://exe.io/WApHIwhE

### ISSUE THREAD

**Issue by dhana-exe** [FastForwardTeam/FastForward#117]
### Domain

https://exe.io/

### Link

NSFW
`
https://exe.io/WApHIwhE
`
### Version

0.0.0

### What browsers are you seeing the problem on?

Chrome

### What OS are you seeing the problem on?

Windows

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by NotAProton** [comment]
```javascript
// Place in ensureDomLoaded
domainBypass(/exey\.io/, () => ifElement("button.btn.btn-primary.btn-goo", a => a.click()))
```

will need crowd bypass for the second step, though.

**Comment by dhana-exe** [comment]
> ```js
> // Place in ensureDomLoaded
> domainBypass(/exey\.io/, () => ifElement("button.btn.btn-primary.btn-goo", a => a.click()))
> ```
> 
> will need crowd bypass for the second step, though.

so i need to just waiting? does the 

// Place in ensureDomLoaded
domainBypass(/exey\.io/, () => ifElement("button.btn.btn-primary.btn-goo", a => a.click()))

will implemented too on next update? or i need to do it alone?

**Comment by NotAProton** [comment]
you don't need to wait too long, I will make a PR and implement it. Meanwhile you can test it

**Comment by dhana-exe** [comment]
> you don't need to wait too long, I will make a PR and implement it. Meanwhile you can test it

ah i see. thanks :D

**Comment by NotAProton** [comment]
Fixed, thanks for the report

**Comment by egypt00king** [comment]
crowd bypass isn't working on exe.io/exey.io

**Comment by egypt00king** [comment]
site addded new domain exey.app
exmple: https://exey.app/9a9HPF8S
need to be bypassed

**Comment by pavan3999** [comment]
exe.io is redirecting to eio.io

**Comment by ghost** [comment]
Will be fixed in mv3, but closed so that all discussion is in #769.
